android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -753,8 +753,8 @@
 		"startBias": ["Avoid [Jungle]", "Avoid [Forest]"],
 		"preferredVictoryType": "Domination",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Great Attila, mighty warrior and leader of the Hunnic empire, your men bow down before you. During the 5th century, you and your brother Bleda have conquered many lands, forged a mighty empire and commanded the most powerful and frightening force of antiquity. You inspired fear to the Roman Emperors themselves, who called you \"Flagellum Dei\", the \"Scourge of God\", as you took kingdom after kingdom in the Balkans, invaded Eastern Roman Empire, Europe into Gaul and Northern Italy, and planned to conquer Rome itself. Your empire didn't survive long after your death, but your name and your legend have, for millenia to come.",
```

Comment:
Please use ' instead of "
Invaded the Eastern Roman Empire, into Gaul (they're all in Europe)